### PR TITLE
Create Speakers “baptistemesta”

### PR DESCRIPTION
**Pull request by noelmace**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by noelmace** [comment]
@baptistemesta I created your profile based on your bio at QCon NYC & DevoxxFR. Could you please tell me quickly if you're good with that, or make some modification? Thx

File: src/pages/speakers/baptistemesta.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  Baptiste travaille dans le développement de logiciels depuis 9 ans. Il a
+  rejoint Bonitasoft dès le début de l'aventure! Il a principalement travaillé
+  sur le moteur d’exécution, qu’il a aidé à créer de toutes pièces. Il a
+  également travaillé sur des outils de développement et des interfaces
+  utilisateurs tout en développant des outils pour aider l'équipe.
+
+twitter: 'https://twitter.com/baptistemesta'
+github: 'https://github.com/baptistemesta'
+linkedin: 'https://www.linkedin.com/in/baptistemesta/'
+previoustalks: >-
```
Comment (baptistemesta) [inline src/pages/speakers/baptistemesta.md]:
```suggestion
previoustalks: >-
* [DeveloperWeek](https://www.developerweek.com/) 2019: [Gradle: One Technology to Build Them All
](https://developerweek2019.sched.com/event/JJ2K)

```

**Comment by baptistemesta** [comment]
LGTM 👍 :shipit:

Only added developerweek talk

**Comment by noelmace** [comment]
Oh, snap, I didn't see that your suggestion broke the build ... I fixed it 5b0a897
Yep, I forgot to enable the outgoing webhooks in Netlify... I fixed it too :sweat_smile:

**Pull request merged**